docs/get-started/sumo-logic-ui.md

```diff
@@ -1,115 +1,143 @@
 
-When you first log in, you'll land on the Sumo Logic Home page. The **Left Nav Bar** menu bar is a one-stop location where you can access the following:
+This page will help you get to know the Sumo Logic user interface (UI).
 
-### Access dashboards and searches
 
-The **Left Nav Bar** provides easy access to libraries, personal collections of dashboards, searches, and folders. Click the icons at the top of the Left Nav Bar (left to right) to view:
 
-* **Recent** dashboards and searches
-* A list of your **Favorites** (dashboards and searches)
-* Your **Personal** library of dashboards and searches,
-* A **Library** of shared dashboards and searches (within your organization) respectively.
+## Using the left navigation panel
+
+The left-side navigation panel is where you can access your library of dashboards as well as all of our services.
+
+### Home
+
+When you first log in, you'll land on the Sumo Logic **Home** page > **Home** tab, which provides an at-a-glance view of the following:
+
+* Recently Opened Dashboards
+* Recently Run Searches
+* Recommended Dashboards 
+* Pinned Searches
 
-![TUI_Left_Nav.png](/img/get-started/ui/TUI_Left_Nav.png)
+<img src={useBaseUrl('img/get-started/dashboard-searches.png')} alt="dashboard-searches.png" />
 
+### Access dashboards and searches
+
+The left nav panel provides easy access to libraries, searches, folders, and your personal collection of dashboards. Click the icons at the top of the left-side navigation panel to view:
+
+* Your **Library**, which contains:
+   * Your **Personal** dashboards and searches.
+   * Your own **Installed Apps** from the App Catalog.
+   * Dashboards and searches shared within your organization.<br/><img src={useBaseUrl('/img/get-started/library.png')} alt="library.png" width="500"/>
+* **Recent** dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/recent.png')} alt="recent.png" width="350"/>
+* Your **Favorites** list of favorited dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/favs.png')} alt="favs.png" width="350"/>
+
+<!--
 ### Search and switch browsing modes
 
-The top of the **left nav bar** is where you can search for content and
+The top of the left-side navigation panel is where you can search for content and
 users and easily switch browsing modes. 
 
-* Searching—Enter text in the **Search** field to quickly find apps, dashboards, searches, and users. 
-* Switching browsing modes—Click the **Details** icon and make a selection from the dropdown menu.
+* Searching. Enter text in the **Search** field to quickly find apps, dashboards, searches, and users. 
+* Switching browsing modes. Click the **Details** icon and make a selection from the dropdown menu.
 
-![TUI_Search_Details.png](/img/get-started/ui/TUI_Search_Details.png)
+![TUI_Search_Details.png](/img/get-started/TUI_Search_Details.png)
+-->
```

Comment:
@agaur It looks like [this functionality](https://help.sumologic.com/docs/get-started/sumo-logic-ui/#searchand-switchbrowsing-modes) doesn't exist in the new UI. Can you confirm?